models/user.model.js

```diff
@@ -1,9 +1,41 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
-const { Array, Boolean, String } = Schema.Types;
+const { Boolean, String } = Schema.Types;
 
 const UserSchema = new Schema({
+<<<<<<< HEAD
```

Comment:
Could we get rid of the merge markers in this file? We may need to rebase since master includes `friends` as well.

Reply:
I think the user model should just be what's currently on master + the `profileImg` field you added here